barcodes_generator_abstract/tests/test_barcodes_generator_abstract.py

```diff
@@ -82,11 +82,6 @@ def test_generate_sequence_sequence(self):
         self.user_fake.generate_barcode()
         self.assertEqual(self.user_fake.barcode, "2000001000007")
 
-        self.user_fake.generate_base()
-        self.assertEqual(self.user_fake.barcode_base, 2)
-        self.user_fake.generate_barcode()
-        self.assertEqual(self.user_fake.barcode, "2000002000006")
-
```

Comment:
Why do you remove this test. It looks valid ?

Reply:
Hi! Because it will be the same base and barcode. With this change if the product already has a base, it will not generate a new one

Reply:
If the code is called two times, It should be fixed. But this change is not desired.

Reply:
I don't undestand really well, it make sense that it doesn't change because you can't change the base two times in UI

Reply:
Maybe I did'nt understood the point. I'll investigate a little. 
Thanks for your time.

Reply:
I just investigated a little. the problem exists.

I permit me to fix it with https://github.com/OCA/stock-logistics-barcode/pull/651.

@dalonsofl : could you review it ? 

thanks !

Reply:
Hi @legalsylvain !

I checked the PR and it looks great!

I change my test like yours and it seems to work great too.

I get the same value with your PR as mine, so both of them are correct.

What do you think with the new changes?

Thanks!!

Reply:
Hi @dalonsofl. Thanks for reviewing the other PR. 
Well, I prefer to have explicit text that explain what does the generate_base() function, and to avoid to double check if barcode_base is defined. 
I so merged the other PR. 

thanks for raising the issue.

Reply:
I agree, the result is the same, so if you think it's better to have comments on generate_base() , I'm fine with that.

Reply:
I merged the other PR, you can close this one.

thanks for the investigation.